src/components/IconTextIcon/IconTextIcon.tsx

```diff
@@ -34,7 +34,12 @@ export const IconTextIcon: React.FC<IconTextIconProps> = ({
                     />
                 )}
 
-                <Typography className="flexFluid" {...typoProps}>{children}</Typography>
+                <Typography
+                    className={clsx('flexFluid')}
+                    {...typoProps}
+                >
```

Comment:
I can add `isNowrap`, but it doesn't seem to do anything. Also as above, the `Typography` width isn't constrained to the cell without using `width: 100%` in the `IconTextIcon` component scss.

Reply:
Ohh you mean set it as default for the `Typography` component inside `IconTextIcon` instead of setting it from outside `IconTextIcon` and passing it in....sure :) that works for me 👍🏾